docs/07-Advanced-Features/LUA-Scripting.md

```diff
@@ -1 +1,4 @@
 # LUA scripting
+
+For a basic introductions see [this wiki section](https://wiki.rusefi.com/Lua-Scripting/)
+For examples see [files in example directory](https://github.com/FOME-Tech/fome-fw/tree/master/firmware/controllers/lua/examples)
```

Comment:
```suggestion

For examples see [files in example directory](https://github.com/FOME-Tech/fome-fw/tree/master/firmware/controllers/lua/examples)
```

Needs an extra line break to insert a line break:
![image](https://github.com/FOME-Tech/wiki/assets/8540239/2d781677-837b-4fcb-96b5-93c3d3927bb0)